rpc_proxy.go

```diff
@@ -365,7 +365,7 @@ func (p *rpcProxy) makeDirector(allowLitRPC bool) func(ctx context.Context,
 		handled, conn, err := p.subServerMgr.GetRemoteConn(requestURI)
 		if err != nil {
 			return outCtx, nil, status.Errorf(
-				codes.Unavailable, err.Error(),
+				codes.Unavailable, err.Error(), nil,
```

Comment:
nit: There's already a `status.Error` function available that won't require the passing of `nil` here, so I would suggest using that instead.